### DISCUSSION TITLE
wait-cond doesn't honour max-time ?

### DISCUSSION THREAD

**Discussion by mmontone** [mdbergmann/cl-gserver#50] [General]
Hello,

I tried wait-cond today, and it doesn't exit for me after :max-time time.

https://github.com/mdbergmann/cl-gserver/blob/7f94927367f003b1d23d85c2e1a9ed461b8bf4bb/src/utils.lisp#L38

Shouldn't it be `(incf wait-acc sleep-time)` there?

**Comment by mdbergmann** [comment]
Now that you say it, I guess it should.
I initially copied this from somewhere (when I didn't understand the loop construct. :)
Thanks for mentioning, I'll fix it.
